.vscode.example/settings.json

```diff
@@ -1,8 +1,3 @@
 {
-  "go.lintTool": "golangci-lint",
```

Comment:
@hookenz `golangci-lint run --help | grep fast` shows the flag exists, the same for `E`.